packages/site-client/src/pages/package/wco-package-page.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {CustomElement} from '@webcomponents/catalog-api/lib/_schema.js';
+import {html, css} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {unsafeHTML} from 'lit/directives/unsafe-html.js';
+
+import {WCOPage} from '../../shared/wco-page.js';
+import '../catalog/wco-element-card.js';
+
+export interface PackageData {
+  name: string;
+  description: string;
+  version: string;
+  elements: CustomElement[];
+}
+
+@customElement('wco-package-page')
+export class WCOPackagePage extends WCOPage {
+  static styles = [
+    WCOPage.styles,
+    css`
+      .full-screen-error {
```

Comment:
We should eventually move this to common styles

Reply:
+1, done